From: Christophe Jaillet <christophe.jaillet@wanadoo.fr>
Date: Mon, 13 Jul 2015 21:51:38 +0200
Subject: [PATCH 51/58] net: qlcnic: Deletion of unnecessary memset
Patch-mainline: v4.3-rc1
Git-commit: e29dd44325e2fe121088094dcb3055c8d9e5202a
References: bsc#922064 FATE#318609

There is no need to memset memory allocated with vzalloc.

Signed-off-by: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
Acked-by: Shahed Shaikh <shahed.shaikh@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Gary Ching-Pang Lin <glin@suse.com>
---
 drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c
@@ -2388,7 +2388,6 @@ int qlcnic_alloc_tx_rings(struct qlcnic_
 			qlcnic_free_tx_rings(adapter);
 			return -ENOMEM;
 		}
-		memset(cmd_buf_arr, 0, TX_BUFF_RINGSIZE(tx_ring));
 		tx_ring->cmd_buf_arr = cmd_buf_arr;
 		spin_lock_init(&tx_ring->tx_clean_lock);
 	}
